utils/account.ts

```diff
@@ -1,7 +1,12 @@
 import { KeyringAccount } from '@/utils/types/types'
 import keyring from '@polkadot/ui-keyring'
 import { getAddress } from '@/utils/extension'
-import { decodeAddress, encodeAddress, addressEq } from '@polkadot/util-crypto'
+import {
+  decodeAddress,
+  encodeAddress,
+  addressEq,
+  addressToEvm,
```

Comment:
didn't knew about this one

Reply:
People were able to claim KSM/DOT from DOT ICO that happened on ETH